inputfiles/addedTypes.jsonc

```diff
@@ -1303,7 +1320,8 @@
                         },
                         "value": {
                             "name": "value",
-                            "overrideType": "T"
+                            "overrideType": "T",
+                            "required": true
```

Comment:
If we make `value: T | undefined`, should we also make it optional? Or does that not work well together with [`exactOptionalPropertyTypes`](https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes)?

Reply:
I don't think so, the spec is pretty clear that the `value` property would exist: https://streams.spec.whatwg.org/#default-reader-read

Reply:
You're right. Reading the spec you've linked to, it seems pretty explicit that the `value` property will be in the object returned from `read`, even from a `ReadableStreamBYOBReader`.